src/errors/test-run/templates.js

```diff
@@ -187,9 +187,15 @@ export default {
     `),
 
     [TYPE.actionIncorrectKeysError]: err => markup(err, `
-         The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
+        The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
 
-         ${err.getCallsiteMarkup()}
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.hybridFnExecutingInterruptedError]: err => markup(err, `
+        Client code execution was interrupted by page unload.
```

Comment:
So, let's compose the final error text.
We should add to the error text something like: `If page reloading triggered by the executing script use testcafe actions for this` (i mean the script in the hybrid function). Also we can add like: `If page reloading is a result of the previous action, handle it with the existing api`
/cc @VasilyStrelyaev @inikulin